docs/contributor/arch/002-fluent-bit-without-file-system-buffer.md

```diff
@@ -18,12 +18,12 @@ After the in-cluster Loki backend has been removed, the pipeline isolation requi
-2. **Split log stream**: Logs are read by a `tail` input plugin. Before the stream is split into a pipeline-specific stream, global filters like the Kubernetes filter are applied; and before logs are written to an output, each stream can have additional filters. Pipelines can be isolated using the persistent buffer. However, it's impossible to pause the input in the situation of a backend outage. The complexity of this setup is high.
-3. **Dedicated log streams**: Each pipeline has its own `tail` input plugin, a list of filters, and an output plugin. This setup isolates the log processing between different pipelines. In the case of a problem, the streams can be paused individually. This setup has medium complexity.
-4. **Dedicated Fluent Bit instances**: Each pipeline gets its own Fluent Bit DaemonSet. This setup isolates also the CPU and memory resources per pipeline with the cost of a larger overhead. This setup has medium complexity.
+- **Single log stream:** Logs are read by a `tail` input plugin, one or more filters are applied, and written to the backend by multiple output plugins. The input cannot be controlled based on the condition of individual outputs. Filters cannot be applied per individual pipeline. The complexity of this setup is low.
+- **Split log stream**: Logs are read by a `tail` input plugin. Before the stream is split into a pipeline-specific stream, global filters like the Kubernetes filter are applied; and before logs are written to an output, each stream can have additional filters. Pipelines can be isolated using the persistent buffer. However, it's impossible to pause the input in the situation of a backend outage. The complexity of this setup is high.
+- **Dedicated log streams**: Each pipeline has its own `tail` input plugin, a list of filters, and an output plugin. This setup isolates the log processing between different pipelines. In the case of a problem, the streams can be paused individually. This setup has medium complexity.
+- **Dedicated Fluent Bit instances**: Each pipeline gets its own Fluent Bit DaemonSet. This setup isolates also the CPU and memory resources per pipeline with the cost of a larger overhead. This setup has medium complexity.
 
-Currently, Telemetry Manager uses option 2. 
+Currently, Telemetry Manager uses option 2.
 Option 1 does not fulfill our requirement to apply individual filters per pipeline. Option 4 causes an unacceptable resource overhead for our typical setup of two pipelines (application logs and access logs). Option 2 and 3 allow log filter settings per pipeline.
 
 We consider option 3 to be the best Fluent Bit configuration for our requirements because of its lower complexity. The throughput of option 3 has shown to be better than option 2 without changing Fluent Bit's CPU and memory limits.
```

Comment:
```suggestion
We consider dedicated log streams to be the best Fluent Bit configuration for our requirements because of its lower complexity. The throughput of dedicated log streams has shown to be better than split log stream without changing Fluent Bit's CPU and memory limits.
```